compiler/src/iree/compiler/Tools/init_mlir_passes.h

```diff
@@ -65,6 +67,22 @@ inline void registerMlirPasses() {
   // Linalg
   registerLinalgPasses();
 
+  // Tosa
+  registerTosaToMLProgram();
```

Comment:
we don't want to do this here - the tosa passes should be registered by the tosa input plugin (compiler/plugins/input/TOSA/PluginRegistration.cpp) - only passes used in all configurations should be here as we try to keep dependencies as light as possible (the compiler already takes ages to build, and any optional component like TOSA needs to be completely controlled by the build flags (in this case `IREE_INPUT_TOSA`).